android/build.gradle

```diff
@@ -3,17 +3,19 @@
 buildscript {
     ext {
         buildToolsVersion = "28.0.3"
-        minSdkVersion = 16
+        minSdkVersion = 18
         compileSdkVersion = 28
         targetSdkVersion = 28
         supportLibVersion = "28.0.0"
+        kotlinVersion = "1.3.0"
```

Comment:
Why did you add kotlin?

Reply:
Requirement by Detox's latest version

Reply:
hmmm I think I will add some comments to these things